static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -344,6 +378,83 @@ function SummaryContent({
     organization.features.includes('performance-spans-new-ui') &&
     organization.features.includes('insights-initial-modules');
 
+  // This is adapted from the `getEventFieldValues` function in `events/searchBar.tsx`
+  const getTransactionFilterTagValues = memoize(
+    (tag, queryString): Promise<string[]> => {
+      const projectIdStrings = eventView.project?.map(String);
+
+      if (isAggregateField(tag.key) || isMeasurement(tag.key)) {
+        // We can't really auto suggest values for aggregate fields
+        // or measurements, so we simply don't
+        return Promise.resolve([]);
+      }
+
+      // device.class is stored as "numbers" in snuba, but we want to suggest high, medium,
+      // and low search filter values because discover maps device.class to these values.
+      if (isDeviceClass(tag.key)) {
+        return Promise.resolve(DEVICE_CLASS_TAG_VALUES);
+      }
+
+      const fetchPromise = fetchTagValues({
```

Comment:
Not totally necessary, but the results should be improved (and return more quickly) if you pass time range parameters with `endpointParams`.

You can also pass `sort: '-count'` for a more useful order (it defaults to last_seen)